web/lti.go

```diff
@@ -41,13 +40,13 @@ func loginWithLti(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	p := utils.NewProvider(ltiConsumerSecret, fmt.Sprintf("%s%s", c.GetSiteURLHeader(), c.Path))
+	p := utils.NewProvider(ltiConsumerSecret, c.GetSiteURLHeader()+c.Path)
 	p.ConsumerKey = ltiConsumerKey
 	if ok, err := p.IsValid(r); err != nil || ok == false {
 		mlog.Error("Invalid LTI request: " + err.Error())
 		c.Err = model.NewAppError("loginWithLti", "api.lti.login.app_error", nil, "", http.StatusNotImplemented)
 		return
 	}
 
-	http.Redirect(w, r, c.GetSiteURLHeader(), http.StatusFound)
+	http.Redirect(w, r, c.GetSiteURLHeader()+"/signup_lti", http.StatusFound)
```

Comment:
Don't hardcode the part suffix. Can this be stored in a separate variable?

Reply:
The current implementation is similar to how mattermost handles redirects elsewhere.